server/@types/express/index.d.ts

```diff
@@ -34,10 +40,12 @@ declare module 'express-session' {
     newGoals: Array<NewGoal> // An array of NewGoal representing the Goals that have been added
     updateGoalForm: UpdateGoalForm
     prisonerListSortOptions: string
-    pageFlowQueue: PageFlowQueue
+    pageFlowHistory: PageFlow
```

Comment:
This allows us to keep a separate list of pages that have been visited (rather than trying to repurpose the `pageFlowQueue` which got messy, especially when it came to the previous work experience!).